Source/OxyPlot.SkiaSharp/OxyPlot.SkiaSharp.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What's up with this?

Reply:
Very good question...
Inspecting the file with a hex editor reveals that dependabot removed the UTF-8 BOM.
I was vaguely aware that these BOMs exist, but never really bothered about them. I checked a couple of other files in the repo, and all of them did also have the BOM at the beginning. So I would suggest to leave the BOM as is, i.e. reverting this change by dependabot.